ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp

```diff
@@ -232,8 +232,7 @@ int main()
 	delete dispersion;
 	delete particleBunch;
 	delete tracker;
-	// delete scatterModel;
-	delete collimateProcess;
```

Comment:
did you mean to remove `delete collimateProcess`? it is initialized with a 'new' on line 194?

Reply:
The tracker takes ownership and does the deletion. Happens in the ProcessStepManager destructor.

(On the TODO list for smart pointers :-) )

Reply:
All good, quite proficient with smart pointers now, btw, happy to help :)